src/platforms/components/TextReward.tsx

```diff
@@ -58,11 +61,13 @@ const SecretTextReward = ({ platform, withMotionImg }: RewardProps) => {
           tooltipLabel:
             platform?.capacity === platform?.claimedCount
               ? "All available rewards have already been claimed"
+              : claimed
```

Comment:
Same here - for the tooltip, you can use `isDisabled` ;)

Reply:
Hmm, this button is a link button and it's label is not tell user what they can do, and it's Tooltip is not conditionally rendered.
So I'm thinking, maybe I'll put "You can reveal the secret" text back to here.

Reply:
Hmm, i figured out another solution, I'll put the claimed variable into the if statement. I tested out and it is working perfectly

Reply:
when it's claimed we don't need to disabled the button, and we can reveal the secret